client/src/components/pages/auth/Login.js

```diff
@@ -110,9 +110,9 @@ export class Login extends Component {
                 />
                 <div className="center">
                     <p>
-                        Don't have an account?{' '}
-                        <Link to="/register" className="button small secondary">
-                            Register
+                        Forgot password?{' '}
+                        <Link to="/login" className="button small secondary">
+                            Reset
```

Comment:
why do we add button for functionality that doesn't exist

Reply:
Because I was getting rid of an old button that was irrelevant, and i figured why not replace it with something that will need to be added in the future anyways